redis_cloud/README.md

```diff
@@ -0,0 +1,101 @@
+# Agent Check: Redis Cloud
+
+## Overview
+
+Redis is a fast, versatile data store that supports a variety of data structures, including strings, hashes, lists, sets, streams, and more. It also offers programmability, extensibility, persistence, clustering, and high availability. The community edition of Redis adds additional data models and capabilities, which include vector search, probabilistic data structures, JSON support, and full-text search.
+
+The [Redis Cloud][1] integration is intended for use with Redis Cloud deployments of Redis software. It is not for use with Redis Enterprise installations. For [Redis Enterprise][2], see the [Datadog Redis Enterprise integration][3].
+
+The integration provides metrics for three critical cluster components: databases, nodes, and shards, via a Datadog Agent. This allows you to monitor database throughput, memory utilization, CPU usage, connection counts, replication health, and a variety of additional metrics within Datadog. You can use this information to understand the overall health of your Redis Cloud clusters, diagnose application performance issues, and prevent downtime.
+
+For a full list of supported metrics, see the **Metrics** section below.
+
+## Setup
+
+### Installation
+
+1. Run the following command to install the Agent integration:
+ - in Datadog v6
+   ```shell
+   datadog-agent integration install -t datadog-redis_cloud==1.0.0
+   ```
+ - in Datadog v7   
+   ```shell
+   agent integration install -t datadog-redis_cloud==1.0.0
+   ```
+   
+2. Configure the integration by setting the `openmetrics_endpoint` to your cluster's master node. See [Getting Started with Integrations][4] for more information.
```

Comment:
```suggestion
2. Configure the integration by setting `openmetrics_endpoint` to your cluster's primary node. See [Getting Started with Integrations][4] for more information.
```

Reply:
we use the term 'master' and not 'primary.